charts/sscs-hearings-api/values.yaml

```diff
@@ -30,3 +30,8 @@ java:
     CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     COMMON_REF_API_URL: "http://rd-commondata-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     JUDICIAL_REF_API_URL: "http://judicial-ref-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    EXUI_API_URL: "https://manage-case.{{ .Values.global.environment }}.platform.hmcts.net"
+    HMC_API_URL: "http://hmc-cft-hearing-service-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    CORE_CASE_DATA_JURISDICTION_ID: SSCS
```

Comment:
Can't we just default these in application.yaml?

Reply:
I was just doing what was done in tribunals:
![image](https://user-images.githubusercontent.com/18538046/163399186-a2c8690f-5879-4291-8cd5-9ed2ee5f21ad.png)
I'll remove them if you want?